rally/challenges/default.json

```diff
@@ -0,0 +1,52 @@
+    {
+      "name": "append-no-conflicts",
+      "description": "Indexes the whole document corpus using Elasticsearch default settings. We only adjust the number of replicas as we benchmark a single node cluster and Rally will only start the benchmark if the cluster turns green. Document ids are unique so all index operations are append only.",
+      "default": true,
+      "schedule": [
+        {
+          "operation": "delete-index"
+        },
+        {
+          "operation": {
+            "operation-type": "create-index",
+            "settings": {{index_settings | default({}) | tojson}}
```

Comment:
I think a short readme file documenting the exposed track parameters would be nice. See e.g. https://github.com/elastic/rally-tracks/tree/master/geonames#parameters